plugins/3scale-backend/package.json

```diff
@@ -41,5 +41,10 @@
     "dist",
     "config.d.ts"
   ],
-  "configSchema": "config.d.ts"
+  "configSchema": "config.d.ts",
+  "repository": "github:janus-idp/backstage-plugins",
+  "keywords": "backstage",
```

Comment:
Keywords should be an array https://nodesource.com/blog/the-basics-of-package-json#thekeywordsproperty

Reply:
thanks, fixed that